server/conf/application.conf

```diff
@@ -308,6 +308,28 @@ play.filters {
   enabled += filters.LoggingFilter
   enabled += filters.ValidAccountFilter
 
+  # GZip filter configuration
+  gzip {
+
+    # The maximum amount of data to send to the Gzip compressor in one go. Use `0` or `2147483647` to disable the buffering.
+    #
+    # In general, it is recommended to turn off the buffer and prevent generation of overlong chunks at the source.
+    bufferSize = 8k
```

Comment:
Please add an allow-list for file types.  Image formats (like png) are already compressed, and shouldn't be double-compressed (they can actually increase in size, and rarely decrease).

```
gzip {

    contentType {
        # If non empty, then a response will only be compressed if its content type is in this list.
        whiteList = [ "text/*", "application/javascript", "application/json" ]
    }
}
```

Reply:
Good call! I'm actually surprised there wasn't a default allowlist along these lines in the framework!